roles/openshift_master_facts/tasks/main.yml

```diff
@@ -6,7 +6,8 @@
     openshift_master_default_subdomain: "{{ osm_default_subdomain | default(None) }}"
   when: openshift_master_default_subdomain is not defined
 
-- fail:
+- name: Verify required variables are set
+  fail:
```

Comment:
Could we please move the commits unrelated to molecule to a separate PR? E.g. adding names to tasks could easily be reviewed and merged independent of molecule, and would also make it easier to read and focus on what is necessary to add the molecule-based tests.